snapcraft/internal/build_providers/_multipass/_multipass_command.py

```diff
@@ -147,21 +166,25 @@ def launch(
         :param str mem: amount of RAM to assign to the launched instance.
         :param str disk: amount of disk space the instance will see.
         :param str remote: the remote server to retrieve the image from.
-        :param str cloud_init_file: path to a user-data cloud-init configuration.
+        :param str cloud_init: path to a user-data cloud-init configuration.
```

Comment:
we do not use cloud init anymore, so feel free to just remove reference to it here

Reply:
ACK

Reply:
Done, removed.